CanvasAPI/CanvasAPI.py

```diff
@@ -151,3 +152,97 @@ def getCoursesAssignmentsObjects(self, courseID):
                 .jsonObjects(object_hook=self.jsonObjectHook)
 
         return coursesAssignments
+
+    def getCoursesUsers(self, courseID, enrollmentType=None, **kwargs):
+        """
+        Get Canvas Users objects as requests Response object.  May be one of multiple pages.
+
+        :param courseID: ID number of the Canvas course object's User Objects to be retrieved
+        :type courseID: int
+        :param enrollmentType:
+        :type enrollmentType: str
+        :return:
+        :rtype: requests.models.Response
+        """
+        assert type(courseID) is int
+
+        if enrollmentType:
+            if not isinstance(enrollmentType, str):
+                raise TypeError('enrollmentType must be string')
+            if 'enrollment_type' not in kwargs:
+                kwargs['enrollment_type'] = str(enrollmentType)
+
+        queryURI = self._QueryURIs.COURSES_USERS.format(courseID=courseID)
+        response = self.get(queryURI, params=kwargs)
+
+        return response
+
+    def getCoursesUsersObjects(self, courseID, enrollmentType=None, **kwargs):
+        """
+        Get Canvas User objects as CanvasObjects parsed from JSON
+
+        :param courseID: ID number of the Canvas Course object to find User objects
+        :type courseID: int
+        :param enrollmentType:
+        :type enrollmentType: str
+        :return: An object representing the Canvas Users contained
+            in the API response, otherwise :class:`None<None>`
+        :rtype: CanvasObject
+        """
+        assert type(courseID) is int
```

Comment:
I liked it at first, too, but after I used it a while, I changed my mind.  It raises an `AssertionError` when the test fails, which is not what I usually want.  It should be a "type" exception instead, so I'm writing these differently now.  At some point, I'll need to update these lines.  This type of assertion is good for unit tests, though.

Also, I learned that `type(obj) is xxx` is not recommended.  We should use `isinstance(obj, xxx)` instead.